bittensor/utils/_register_cuda.py

```diff
@@ -57,20 +57,19 @@ def _hex_bytes_to_u8_list(hex_bytes: bytes):
         ]
         return hex_chunks
 
-    def _create_seal_hash(block_and_hotkey_hash_hex: bytes, nonce: int) -> bytes:
+    def _create_seal_hash(block_and_hotkey_hash_hex_: bytes, nonce: int) -> bytes:

```

Comment:
Not sure the trailing underscore is needed here

Reply:
block_and_hotkey_hash_hex used outside the scope of _create_seal_hash function